src/olympia/api/tests/test_jwt_auth.py

```diff
@@ -146,7 +160,6 @@ def test_expired_token(self):
         api_key = self.create_api_key(self.user)
         payload = self.auth_token_payload(self.user, api_key.key)
         payload['exp'] = (datetime.utcnow() -
-                          settings.JWT_AUTH['JWT_EXPIRATION_DELTA'] -
```

Comment:
It should be, since it's setting an expiration that is in the past anyway ? I wasn't sure why it was there, and tests pass